api/src/MemberMatch/ProbabilityProvider.cs

```diff
@@ -6,17 +6,11 @@ namespace RaceResults.MemberMatch
 {
     public class ProbabilityProvider
     {
-        private ICosmosDbContainerClient<MemberMatchRecord> containerClient;
+        private ContainerClient<MemberMatchRecord> containerClient;
```

Comment:
I'm assuming you made these changes because of MemberMatchRecord now implementing IModel. If we're reverting that change, I think we'll have to revert this too.

Reply:
Other way around. This project didnt compile without these changes

Reply:
ICosmosDbContainerClient no longer exists

Reply:
I backed this out